graylog-storage-opensearch2/src/main/java/org/graylog/storage/opensearch2/views/searchtypes/pivot/OSPivotBucketSpecHandler.java

```diff
@@ -61,18 +62,22 @@ the numeric value of it, so that we can sort on it numerically. Any metric aggre
                     if (sortSpec instanceof SeriesSort) {
                         final Optional<SeriesSpec> matchingSeriesSpec = pivot.series()
                                 .stream()
-                                .filter(series -> series.literal().equals(sortSpec.field()))
+                                .filter(series ->
+                                        series.literal().equals(sortSpec.field())
+                                                || (series instanceof Percentile && sortSpec.field().equals(series.id()))
```

Comment:
It looks that we only have one case right now? But would it make sense to introduce a marker interface instead of using Percentile directly?